### PR TITLE
feat: add container security context support

### LINKED ISSUE #973: Implement container-scoped securityContext configuration
Hi, this request is similar to #904; however, in this case we'd like to have the container-scoped securityContext configurable, and passed to the spawned pods.   

This would allow us to set capabilities for the spawned containers (such as NET_ADMIN), which would in turn enable us to run tc/netem to induce impaired networking conditions in the client browser test environment.

https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.14/#securitycontext-v1-core

Please let me know if you'd like further elaboration.

Thanks,
Martin

### PR DESCRIPTION
**Pull request by arnaud-deprez**
Closes #973

### Description
Should fix #973 

### Motivation and Context
Allow to define [Container Security Context](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.15/#securitycontext-v1-core) for both hub and selenium grid. 
Attention the same value is reused for both Pods.

### How Has This Been Tested?
Unit tests and manual integration tests

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Comment by arnaud-deprez** [comment]
Hi @davidcollom, @diemol, @martincfo,

Sorry for the delay, I was quite busy these days.

I did a change in #995 that is equivalent to #904 but for `Container Security Context`.
So in short, this allows to configure `Container Security Context` for both `hub` and `grid` but it does not allow to use different context/configuration.

In order to do that, I think we should foresee deeper/heavier changes. While it is still possible to implement such a feature as it is, I feel the chart and the number of environment variables to configure it all starts getting too heavy and confusing.
For example for `Pod Security Context` or `Container Security Context`, we should allow user to set json value in environment variable as it is impossible to guess all combination.

So instead of adding again some environment variable, I was thinking about using a configuration file which will look like a lot to `Pod` for the grid.

However, this idea is not clear yet in my mind as there are a lot of environment variables used for docker and docker swarm setup and so we will need to define what takes precedence over what or review the whole thing.

So basically, I propose to integrate this change as-is and then we can think about a better design/solution to fully customise zalenium deployment.

What do you think ?

**Comment by diemol** [comment]
I think so far it looks good @arnaud-deprez, thanks!
If we need something more granunar in the future, then we could revise this.
